src/ValidatorVisitor.php

```diff
@@ -358,6 +362,10 @@ public function leaveNode(Node $node){
                     } else {
                         $this->sandbox->validationError("Sandboxed code attempted use invalid namespace or alias!", Error::DEFINE_ALIAS_ERROR, $node);
                     }
+                    if ($this->sandbox->isWhitelistedInterface($use->alias))
```

Comment:
Sorry, what does "CS :s" mean?
Also, I think I still have some flawed logic in this, I will work on this again and update the PR later.